environment.yml

```diff
@@ -0,0 +1,64 @@
+name: volunteer-atlas
```

Comment:
Is there a reason we're adding a conda environment file? Maybe add this file to the .gitignore?

Reply:
I created an anaconda environment for myself, and figured it could be useful to add to the repo in case future others might want to do so as well. 
If you prefer to not keep it in, I think you should be able to remove it from this PR and I will add it to my .gitignore for the future

Reply:
I'm just wondering if would conflict with the requirements.txt file that's required for deploying to heroku. I've never had both in the same repo. Was it hard to get an anaconda environment setup from the requirements.txt file?

Reply:
hm good point, I have no experience with heroku so I cant say. 
For the anaconda environment file its essentially just a reformatting of the requirements.txt file, so it was pretty easy to create. I think, to be safe, I'll remove it for now then

Reply:
Awesome, that was the only thing I noticed. I'm just worried that if we add dependencies in the future, it'd be better to have one source of truth rather than two.